src/state.rs

```diff
@@ -106,12 +106,10 @@ impl SpriteSheetAnimationState {
 
             self.elapsed_in_frame -= frame.duration;
             frame = animation.frames[self.current_frame];
-            sprite.index = frame.index;
```

Comment:
I just realized that removing this will lead to weirdness of `Once` animations that skip several frames, since the final frame might not get set before returning. Will update.